internal/rules/policies/kubernetes/policies/advanced/default_namespace_should_not_be_used.rego

```diff
@@ -0,0 +1,33 @@
+# METADATA
+# title: "The default namespace should not be used"
+# description: "ensure that default namespace should not be used"
+# scope: package
+# schemas:
+# - input: schema.input
+# related_resources:
+# - https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/
+# custom:
+#   id: KSV110
+#   avd_id: AVD-KSV-0110
+#   severity: LOW
+#   short_code: default-namespace-should-not-be-inuse
```

Comment:
`inuse` => `in-use` but actually for consistency with title better to say `used`